web/.eslintrc.precommit.json

```diff
@@ -23,6 +23,7 @@
         "quotes": ["error", "double", { "avoidEscape": true }],
         "semi": ["error", "always"],
         "@typescript-eslint/ban-ts-comment": "off",
-        "sonarjs/cognitive-complexity": ["error", 9]
+        "sonarjs/cognitive-complexity": ["error", 9],
+        "sonarjs/no-nested-template-literals": "off"
```

Comment:
I _hated_ doing this, but six of the files I changed had nested template literals _with_ `msg()` strings embedded, which makes them more or less immobile.  Frustrating as heck.